docs/source/gettingstarted.rst

```diff
@@ -125,18 +135,23 @@ it's a good place to start. Another thing to note is that when fixing,
 you must always be specific about which rules you wish to fix. This
 is to minimise any unintended consequences from making large scale
 changes to your code. In this case we want to try and fix rules
-*L003*, *L009* and *L010*.
+*L003*, *L009*, *L010*, *L034*, *L036* and *L039*.
 
 .. code-block:: bash
 
-    $ sqlfluff fix test.sql --rules L003,L009,L010
+    $ sqlfluff fix test.sql --rules L003,L009,L010,L034,L036,L039
```

Comment:
The `--rules` argument is no longer compulsory for `fix`. I wonder if we should either:
- remove the reference to `--rules` here, and instead only demo it in the later section (line 219).
- adjust the narrative here so we don't fix *all* of the rules but intentionally only choose a subset to fix. In that case I think the later example _shouldn't_ use `--rules` just to show that it's not necessary anymore.

@linchun3 - which do you think is more helpful for a new user reading this?

Reply:
hey @alanmcruickshank thanks for raising this up! I think the 2nd approach of adjusting the narrative (fixing subset followed by all) would be clearer to the reader that `--rules` is optional. I'll update this PR with that.

Reply:
Updated the narrative for the documentation to indicate optional use of `--rules`